tutorials/functional-full-stack/purescript-bridge/tutorial.md

```diff
@@ -0,0 +1,339 @@
+---
+title: Connecting a Haskell Backend to a Purescript Frontend
```

Comment:
@javcasas According to the [official page](http://www.purescript.org), the name is written as PureScript.